plugins/destination/postgresql/client/migrate.go

```diff
@@ -420,7 +420,7 @@ func (c *Client) createPerformanceIndexes(ctx context.Context, table *schema.Tab
 		return nil
 	}
 
-	indexName := "cq_performance_idx"
+	indexName := table.Name + "_cqpi"
```

Comment:
This has an issue when the table name is 62 characters (or whatever the maximum length for an object name), we cannot add 5 more characters to the name. To make the fix backwards compatible we can just use a hash of the entire table name if the length is the max length

Reply:
This is how we name PKs so only following that convention (it will fail for them too, see https://github.com/cloudquery/cloudquery-private/pull/5379

Reply:
Great point! existing PKs use `table_name + '_cqpk'` which is the same number of characters as `_cqpi` so no new issue here.